socprint.sh

```diff
@@ -106,6 +106,10 @@ SOURCE
 CONTRIBUTORS
   Donald Lee, Julius Nugroho, Sean Ng
 
+KEYWORDS
+  how to print NUS SoC National University of Singapore School of Computing
+  printers shell command line basement sunfire mac windows linux bsd ben
+
```

Comment:
```suggestion
amateur alien pov belle delphine cosplay mature bisexual asmr femdom
```
https://www.pornhub.com/insights/2019-year-in-review#searches

Reply:
fantastic work @juxd